test/wasm_worker/proxied_function.js

```diff
@@ -1,7 +1,9 @@
 addToLibrary({
-  proxied_js_function__proxy: 'sync',
+  // Use a non-standaed __proxy flag here.
+  // This is used, for example, in library_html5_webgl.js. With WASM_WORKERS
+  // we want the build to succeed rather than complain about the invalid
+  // proxy type.
+  proxied_js_function__proxy: 'sync_on_webgl_context_handle_thread',
```

Comment:
Is this just a garbage name? Then perhaps `xxx_garbage` or such, to make that more obvious? ("nonstandard" suggests maybe it has meaning, even if not standard)

Reply:
Done